app.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Good job. I think you can also add the heroku Url to the readme for easy access. 

Were we all added as contributors to the app?

Reply:
> Good job. I think you can also add the heroku Url to the readme for easy access.

I will do that now.

> Were we all added as contributors to the app?

Will also do this.

Reply:
@Chiazokam I have added the `url` to the readme and added everyone as contributors.